mia/medium/super-famicom.cpp

```diff
@@ -255,6 +255,7 @@ auto SuperFamicom::analyze(std::vector<u8>& rom) -> string {
   s +={"  board:    ", board(), "\n"};
 
   auto board = nall::split(this->board().trimRight("#A", 1L), "-");
+  if(board.size() <= 1) { board.resize(2); }
```

Comment:
```suggestion
  board.resize(2);
```

The if is not necessary, and we want to settle on a standard pattern.

Reply:
This won't work. The minimum number of entries needs to be two. But there could be more and if we blindly resize to two we will throw away information we need. For example, a perfectly valid board string prior to splitting could read:

LOROM-SA1-RAM-EPSONRTC

Reply:
ah right, the split doesn't have a maxsize which is why i hadn't noticed this case before. It's good as is then